packages/datadog_flutter_plugin/integration_test_app/lib/auto_integration_scenarios/rum_auto_instrumentation_scenario.dart

```diff
@@ -18,7 +18,7 @@ class RumAutoInstrumentationScenario extends StatefulWidget {
 class _RumAutoInstrumentationScenarioState
     extends State<RumAutoInstrumentationScenario> {
   final images = [
-    'https://placekitten.com/300/300',
+    'https://imgix.datadoghq.com/img/about/presskit/kit/press_kit.png',
```

Comment:
why do we reference the same image twice?

Reply:
I had the same question but assumed it wasn't relevant for the scenario for the 2 images to be the same.

Reply:
Yeah the image isn't relevant to the test, and for web you have to grab an image from a server that has its CORS headers set, which placekitten doesn't.